code/game/objects/structures/flora.dm

```diff
@@ -73,14 +73,106 @@
 	..()
 	icon_state = "snowbush[rand(1, 6)]"
 
+
+
+
+//Potted Plants
 /obj/structure/flora/pottedplant
 	name = "potted plant"
 	icon = 'icons/obj/plants.dmi'
 	icon_state = "plant-26"
+	var/dead = 0
+	var/obj/item/stored_item
 	layer = ABOVE_MOB_LAYER
 
-//newbushes
 
+/obj/structure/flora/pottedplant/Destroy()
+	QDEL_NULL(stored_item)
+	return ..()
+
+/obj/structure/flora/pottedplant/proc/death()
+	if (!dead)
+		icon_state = "plant-dead"
+		name = "dead [name]"
+		desc = "It looks dead."
+		dead = 1
```

Comment:
https://github.com/discordia-space/CEV-Eris/blob/master/CONTRIBUTING.md
Should be `TRUE/FALSE` instead `1/0`

Reply:
done.
I was never shown this document before, btw. I will read it and endeavour to follow its guidelines